usaspending_api/database_scripts/matview_generator/summary_transaction_geo_view.json

```diff
@@ -36,13 +36,19 @@
     "  SAA.abbreviation AS awarding_subtier_agency_abbreviation,",
     "  SFA.abbreviation AS funding_subtier_agency_abbreviation,",
     "",
+    "  SUM(COALESCE(CASE",
+    "    WHEN awards.category = 'loans' THEN awards.total_subsidy_cost",
```

Comment:
I don't know if `awards.category = 'loans'` is the best check considering when we're filtering on awards/subawards we're allowing awards.category to be null. If it's guaranteed to be populated based on the award types then I'm fine with it. Just making sure.

Reply:
Good question. According to @nmonga91 this is an acceptable check for pulling in `total_subsidy_cost`.

It helps this case statement is all-encompassing. Either the record is known to be a loan, otherwise use `federal_action_obligation`

Reply:
Good for me.